### ISSUE TITLE
Some stats links for buckets are broken

### ISSUE THREAD

**Issue by edgarcosta** [LMFDB/lmfdb#3622]
There are various stats links that are broken, when the bucket involves ranges:
e.g.:
https://www.lmfdb.org/ModularForm/GL2/Q/holomorphic/?char_order=1&dim={u%27$lte%27:%2010,%20u%27$gte%27:%206}&dim_type=rel
https://www.lmfdb.org/Variety/Abelian/Fq/?g=3&q=11&hyp_cnt={u%27$gte%27:%2017}

However, there are other buckets that don't suffer from this problem, e.g., the first table in
https://www.lmfdb.org/ModularForm/GL2/Q/holomorphic/stats

The problem is probably a missing query formater (see line 1192 of lmfdb/classical_modular_forms/main.py). 
```
def rel_dim_formatter(x):
    return 'dim=%s&dim_type=rel' % x
```

**Comment by AndrewVSutherland** [comment]
Looks good.  While testing I noticed a bug in the genus 2 stats (which has been there a long time and is not related to the recent changes), none of the statistics links for automorphism groups work (because they are including the latex wrapper).  I'll submit a PR shortly to fix this.

https://www.lmfdb.org/Genus2Curve/Q/?geom_aut_grp_id=\(C_2\)

**Comment by AndrewVSutherland** [comment]
Resolved by #3623 (and the related issue noted above is fixed by #3625).
